Python/Object_detection/readme.md

```diff
@@ -0,0 +1,31 @@
+# Object_Detection
+
+It solves the issue #982 .
```

Comment:
Hey there, I will have a thorough look at the Notebook asap. But this is not how we want the [README](https://github.com/HarshCasper/Rotten-Scripts/tree/master/Python/Github_Traffic) to look.